spec/routes/redirections_spec.rb

```diff
@@ -0,0 +1,45 @@
+require "rails_helper"
+require "decidim/dev/test/spec_helper"
+require "decidim/core/test/factories"
+require "decidim/proposals/test/factories"
+
+describe "routing redirections", type: :request do
```

Comment:
Type should be routing. Although if the folder was named `routing` you wouldn't need to specify iy.